src/pages/api/secure/send-reset-pin-link.ts

```diff
@@ -0,0 +1,63 @@
+import jwt from 'jsonwebtoken'
+import { NextApiRequest, NextApiResponse } from 'next'
+
+import { withSessionRoute } from '@/ironAuth/withSessionApiRoute'
+import { appUrl } from '@/utils/constants'
+import { sendResetPinEmail } from '@/utils/email_helper'
+
+// JWT_SECRET must be set in environment variables
+const JWT_SECRET = process.env.JWT_SECRET
+const TOKEN_EXPIRY = '5m'
+
+if (!JWT_SECRET) {
+  throw new Error('JWT_SECRET environment variable is required')
```

Comment:
A top-level Error would crash the app in the instance that this doesn't exist. Instead, add this check to the api handler